GeoJSON/geojson-encoding-rule.md

```diff
@@ -0,0 +1,270 @@
+
+NOTE The INSPIRE Registry can provide the JSON representation of the code list and its entries.
+
+### Properties
+
+Property names remain as they are.
+
+NOTE Namespace prefixes, as used in the default encoding, are not used in the GeoJSON encoding rule.
+
+#### Properties with a `uom` attribute
+
+The unit of measurement attribute (`uom`) on any property `x` has to be retained. It is transformed to a new property of the type with the name `x.uom`.
+
+#### Arrays
+
+Property types for properties with a cardinality greater than `1` and a simple property type (e.g. String, Integer, Float, ...) may use arrays of these simple types. More information is available in the [Extract Primitive Array](../model-transformation/ExtractPrimitiveArray.md) transformation rule.
+
+#### Voidable
+
+The stereotype `<<voidable>>` is not converted. If a property has no value, it may be dropped.
+
+### Association Roles
+
+Association roles are retained as they are, if they are not transformed using a profile-specific rule, such as [Inline associated or aggregated components using type names](../model-transformation/AssociatedComponentsHardType).
+
+## Instance Encoding Rules
+
+This section describes how the encoding is derived from the converted conceptual model, and describes which common rules have to be applied for this encoding rule.
+
+### Common Rules
+
+* `GEOJSON-REQ-01`: The character encoding of all data encoding in GeoJSON shall be UTF-8.
+* `GEOJSON-REQ-02`: As per the requirements of the GeoJSON - IETF RFC 7946 specification, the default CRS for any data set delivered using this encoding rule is a geographic coordinate reference system, using the World Geodetic System 1984 (WGS 84) datum, with longitude and latitude units of decimal degrees, unless there is prior arrangement. This is equivalent to the coordinate reference system identified by the Open Geospatial Consortium (OGC) URN `urn:ogc:def:crs:OGC::CRS84`.
+
+NOTE As INSPIRE mandates the use of the European Terrestrial Reference System 1989  (ETRS89, see [Requirement 1](https://inspire.ec.europa.eu/reports/ImplementingRules/DataSpecifications/INSPIRE_Specification_CRS_v2.0.pdf)) for the areas within the geographical scope of ETRS89 and both CRS84 and ETRS89 use the GRS 80 ellipsoid (although with minor enhancements), we shall assume CRS84 to be equivalent to ETRS89. If, for any dataset, this assumption would be problematic, then GeoJSON cannot serve as an alternative encoding rule for that dataset.
+
+* `GEOJSON-REQ-03`: In the GeoJSON encoding rule, `nilReason` information shall not be maintained per feature, but rather in the dataset metadata. Properties that have `nil` values shall thus be ignored in the encoding process.
+
+NOTE If, for any dataset, there is specific `nilReason` information per feature, then GeoJSON cannot serve as an alternative encoding rule for that dataset
+
+### Alternate Coordinate Reference System support
```

Comment:
Alternative ?

Reply:
Yeah, I was wondering about that one too... @nmtoken?

Reply:
I think it should be alternative, because it is indication of optionality, not of alternation.

Reply:
OK, done.